superset/db_engine_specs/base.py

```diff
@@ -177,6 +178,35 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
         ),
     }
 
+    @classmethod
+    def get_dbapi_exception_mapping(cls) -> Dict[Type[Exception], Type[Exception]]:
+        """
+        Each engine can implement and converge it's own specific exceptions into
```

Comment:
Grammar nit: "its" not "it's"

Reply:
Fixed, yes that's a good point, a simple way of doing it is:

```
if not new_exception:
    return SupersetDBAPIError(str(exception))
```

But can have broader implications, I want to place the base for this, and just interfere on the defined exceptions for Elasticsearch and clickhouse (for now).